test/test_types_plugin.py

```diff
@@ -76,7 +76,7 @@ def test_float_modify_and_query(self):
 
         # Match in range
         out = self.list(u'myfloat:-10..0')
-        self.assertIn(b'aaa', out)
+        self.assertIn('aaa', out)
```

Comment:
Please have a strong feeling :) hehe.  If we continue to use something that works like StringIO, then it's gonna be native strings.